### DISCUSSION TITLE
Login As Button | Use API to Login As User

### DISCUSSION THREAD

**Discussion by KamranBiglari** [drakkan/sftpgo#1825] [Ideas]
Is it possible to use admin token to login as client in Client UI and how it's possible to have this feature?
Is it possible to have a login as button next to each user to login as client into client UI?

**Comment by drakkan** [comment]
Thank you for using SFTPGo at your company, appreciated, but please stop asking the same question over and over again. 
You are using an open source software for free, you cannot expect support guarantee. This is quite clear in the repo [README](https://github.com/drakkan/sftpgo?tab=readme-ov-file#support).

The obvious answer is: "you have the code, make this change and make it available to everyone as per the SFTPGo license", we can then decide whatever includes your changes in the upstream version or not. Also read [this](https://sftpgo.com/compliance.html). Thank you
